routes/users.js

```diff
@@ -26,8 +26,16 @@ router.post("/update-users", (req, res) => {
         const data = projectManager.data;
         // Check if data exists
         if (data) {
-            data.users = req.body.users;
-            res.json({ success: true });
+            const newUsers = [];
+            req.body.users.forEach((user) => {
+                newUsers.push(User.parseFromJson(user));
+            });
```

Comment:
This can also be replaced with a `.map` call. See my previous comment for an example